components/molecule/thumbnail/src/index.scss

```diff
@@ -0,0 +1,116 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bgc-molecule-thumbnail-basic: $c-white !default;
+
+$sizes-molecule-thumbnail: (
+  large: 144px,
+  medium: 72px,
+  small: 48px,
+  xsmall:  40px
+) !default;
+
+$sizes-mobile-molecule-thumbnail: (
+  large: 96px,
+  medium: 72px,
+  small: 40px,
+  xsmall:  32px
+) !default;
+
+$ratios-molecule-thumbnail: (
+  1: 1,
+  4: 3,
+  16: 9
+) !default;
+
+$bd-outer-molecule-thumbnail: 1px solid color-variation($c-gray, 3) !default;
+$bdc-outer-hover-molecule-thumbnail: $c-text-link !default;
+$bd-inner-molecule-thumbnail: 3px solid $c-white !default;
+$bgc-molecule-thumbnail: $c-gray-light !default; 
+$bs-molecule-thumbnail: none !default;
+
+.sui-MoleculeThumbnail {
+  background: $bgc-molecule-thumbnail-basic;
+  border: $bd-outer-molecule-thumbnail;
+  box-shadow: $bs-molecule-thumbnail;  
+  display: inline-block;
+  margin: 0;  
+  position: relative;
+  width: 100%;
+
+  .sui-MoleculeThumbnail-link & {
+    &:hover {
+      border-color: $bdc-outer-hover-molecule-thumbnail;
+    }
+  }
+
+  .sui-ImagePlaceholder-image {
+    height: 100%;
+    object-fit: cover;
```

Comment:
object-fit is not supported by IE11. Do we have some kind of callback ?
https://caniuse.com/#search=object-fit

Reply:
👍